railties/lib/rails/generators/rails/app/templates/config/initializers/enable_yjit.rb.tt

```diff
@@ -0,0 +1,11 @@
+# Automatically enable YJIT as of Ruby 3.3, as it bring very
```

Comment:
@byroot  I could be missing something, but this file and the next look identical to me. Do you only need one of them?

(If you only need one, the other one has a better comment so I vote that one.)

Reply:
🤦 I messed up. Wanted to `mv`, but ended up `cp`...